watchtower/lookout/justice_descriptor_test.go

```diff
@@ -170,20 +195,44 @@ func TestJusticeDescriptor(t *testing.T) {
 				},
 			},
 		},
-		TxOut: []*wire.TxOut{
-			{
+	}
 
-				Value:    int64(sweepAmt),
-				PkScript: justiceKit.SweepAddress,
-			},
-			{
+	if blobType.Has(blob.FlagReward) {
```

Comment:
More or less the same block of code used in `assembleJusticeTxn`, I think we can consolidate a bit.